dev/com.ibm.ws.jaxws.clientcontainer.security/bnd.bnd

```diff
@@ -17,7 +17,8 @@ Bundle-Description: IBM JAX-WS ClientContainer Security; version=${bVersion}
 
 Import-Package: \
   !com.ibm.ws.jaxws.clientcontainer.security, \
-  org.apache.cxf.*;version="[2.6.2,2.6.3)", \
+  org.apache.cxf.*;version="[2.6.2,3.5)", \
+  com.ibm.ws.jaxws.security;version="[1,3)", \
```

Comment:
`[1,3)` seems odd.  Should be be `[2.3,3)?

Reply:
I agree, I can go back and update `com.ibm.ws.jaxws.common` and change it's export version number, but it's actually because that bundle is exporting without a 2.2 version, and as you know that means it defaults to 1.0.